src/components/Feed/ActivityVersions/ActivityVersions.styled.js

```diff
@@ -58,7 +55,8 @@ export const Title = styled.div`
 
 export const Thumbnail = styled(ThumbnailSimple)`
   width: 74px;
-  height: 100%;
+  min-width: 74px;
+  height: unset;
```

Comment:
Technically without the comment now being next to the thumbnail this could be reverted again. But it may still be helpful for cases where e.g. product name is super long and with the "date" inline after it that the thumbnail for whatever reason would become super narrow - with this it'd now remain consistent size.